app/components/category-form/component.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import { categoryValidations } from 'bsc-ember/validations/category';
+import Changeset from 'ember-changeset';
+import lookupValidator from 'ember-changeset-validations';
+
+const { Component,
```

Comment:
keep with one style, or use prettier plugin for visual studio code (I've tested it during the weekend and works nicely)